metrics/web/lib/common/presentation/widgets/metrics_user_menu.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+import 'package:metrics/base/presentation/widgets/base_popup.dart';
+import 'package:metrics/base/presentation/widgets/hand_cursor.dart';
+import 'package:metrics/common/presentation/routes/observers/overlay_entry_route_observer.dart';
+import 'package:metrics/common/presentation/strings/common_strings.dart';
+import 'package:metrics/common/presentation/widgets/metrics_user_menu_card.dart';
+
+/// A widget that displayed the user menu pop-up.
+class MetricsUserMenu extends StatelessWidget {
```

Comment:
```suggestion
class MetricsUserMenuButton extends StatelessWidget {
```
And maybe rename Card -> just Menu?
As this one only opens the menu really so if we change the naming it could more clear that this is a button that triggers menu display.